components/chart/index.tsx

```diff
@@ -20,6 +21,9 @@ function Chart({
   const rootContent = data[0];
   const [zoomLevel, setZoomLevel] = useState(100);
   const [selectedModules, setSelectedModules] = useState([]);
+  const [Description, setDescription] = useState([]);
+  const [name, setName] = useState([]);
+  const [drawerOpen, setDrawerOpen] = useState(false);
```

Comment:
I think no need to use this one, if a description is set that means we can show the description section, if it's not defined or null then we can hide it

Reply:
done